packages/react-slider/src/components/Slider/useSlider.tsx

```diff
@@ -0,0 +1,265 @@
+import * as React from 'react';
+import { makeMergeProps, resolveShorthandProps, useId, useControllableValue } from '@fluentui/react-utilities';
+import {
+  getCode,
+  ArrowDownKey,
+  ArrowUpKey,
+  ArrowLeftKey,
+  ArrowRightKey,
+  PageDownKey,
+  PageUpKey,
+  HomeKey,
+  EndKey,
+} from '@fluentui/keyboard-key';
+import { on } from '@fluentui/utilities';
+import { SliderProps, SliderShorthandProps, SliderState, SliderPublicRef } from './Slider.types';
+import { useMount } from '@fluentui/react-hooks';
```

Comment:
Similar comment as above. We should not be using `@fluentui/react-hooks` in converged components. You might want to copy the implementation of `useMount` into `@fluentui/react-utilities` and importing this from there instead.

Reply:
I created a [pull request](https://github.com/microsoft/fluentui/pull/19196) for adding the `useMount` and `useUnmount` hooks to **react-utilities**